engines/bops_config/app/views/bops_config/application_types/features/edit.html.erb

```diff
@@ -21,6 +21,8 @@
 
       <%= form.fields_for :features, @application_type.features do |ff| %>
         <%= ff.govuk_fieldset legend: {text: t("application_type_features.legends.check_application_details"), size: "s"} do %>
```

Comment:
I wonder if this list might be better generated by iterating over an array of symbols, if we create a method on `ApplicationType` that returns all the available features.

Reply:
Yeah it's the same boilerplate for each option so iterating makes sense here